bokeh/plotting/_graph.py

```diff
@@ -101,11 +101,10 @@ def get_graph_kwargs(node_source, edge_source, **kwargs):
     else:
         hnode_visuals = None
 
-    if any(x.startswith('node_muted_') for x in kwargs):
-        mnode_visuals = pop_visuals(marker_type, kwargs, prefix="node_muted_", defaults=node_visuals)
-    else:
-        mnode_visuals = None
+    #Always set muted glyph
```

Comment:
(here and below) We always lead a space in comments, so 
```suggestion
    # Always set muted glyph
```
But I also would not object if these comments were just removed. As-is they restate the "what" but don't add a "why"

Reply:
I went ahead and removed them in 3bf013b